website/src/pages/dmn-viewer.module.css

```
@@ -0,0 +1,66 @@
+.loadingFallback {
+    padding: 16px;
+}
+
+.scroller {
+    flex: 1;
+    min-height: 0;
+    overflow: auto;
+    padding: 16px;
+}
+
+.canvas :global(.tjs-container) {
```

Stylelint (17.10.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)